### PR TITLE
Setup/favorites page

### PR DESCRIPTION
**Pull request by Jlavarine**
# Description
- Install Router
- Create FavoritesView Page with Router URL
- Display favorited art pieces
- Add ID and default values to art properties

## Issues

Closes #
#1 
#2 
#3 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

**Review by DanielN88 — APPROVED** [review]
Everything looks good, no additional edits needed. Ready for merge.

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
 import './App.css';
+import { Route } from 'react-router-dom';
 
 function App() {
 
   return (
     <main className="App">
-      <Nav />
-      <Container />
+      <Route exact path='/' render={() =>  {
```
Comment (whiteheadol) [inline src/App.js:13]: Heck yeah Router!

File: src/Components/FavoritesView.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+const FavoritesView = () => {
+  const favoriteArt  = useSelector((state) => state.favoriteArt);
+  const allFavoriteCards = favoriteArt.map(art => {
+    return (
```
Comment (nbalmoori) [inline src/Components/FavoritesView.js:6]: Concise way of having this display the cards

**Review by nbalmoori — APPROVED** [review]
Looks great!

**Review by whiteheadol — APPROVED** [review]

**Pull request merged**